Automattic/wp-calypso
# Janitorial: Refactor build-languages script(s)

## Proposed Changes

This deduplicates the `build-languages.js` script that was duplicated in 3 places with virtually no differences into a single script runnable as a command. My rationale in doing this is that I need to replicate this file once more for `verbum-block-editor` and felt uneasy about having 4 copies of the same script. So I made this easy-to-review PR as a first step.

## Testing Instructions

1. **Don't check out this branch and stay on trunk**.
2. Go to `apps/command-palette-wp-admin`. 
3. Run `yarn build && yarn translate`. 
4. Run `ls -l` in `apps/command-palette-wp-admin/dist/languages`. Save the output to a text file somewhere.

Repeat the same operation exactly for `apps/blaze-dashboard` and `apps/odyssey-stats`.

Now, checkout this branch and:

1. Run `yarn` to update `calypso-build` script. If the build gets stuck, delete `packages/components/dist` folder.
2. Repeat the instructions ablove.
3. The output of the `ls -l` command should not change between trunk and here, besides the modification date.
